src/helpers/index.js

```diff
@@ -5,7 +5,6 @@ export { default as delegateEvent } from './delegateEvent.js';
 export { default as getDataFromHtml } from './getDataFromHtml.js';
 export { default as fetch } from './fetch.js';
 export { default as getCurrentUrl } from './getCurrentUrl.js';
-export { default as normalizeUrl } from './normalizeUrl.js';
+export { default as Location } from './Location.js';
```

Comment:
Something I wanted to ask before already: Why are we not simply importing the defaults like this?

```js
export Location from './Location.js';
```

Reply:
I don't think that's valid syntax. Just checked and it errors out: `Declaration or statement expected`

Reply:
Turns out your example is [a proposed syntax](https://github.com/tc39/proposal-export-default-from) and there's [a babel plugin](https://babeljs.io/docs/en/babel-plugin-proposal-export-default-from) for it.

Reply:
Ooops! As you might have noticed in my comment ("[...]*importing* the defaults[...]"), I totally overlooked that these are `export`, not `import` statements. I saw an import using the `{ default as x }` syntax [over here](https://github.com/swup/swup/blob/next/src/helpers/createHistoryRecord.js#L1) the other day. That's where I was confused. But still good to know that my example is a proposed syntax! 😄